wordpress-mobile/WordPress-iOS
# Issue/13063 insert captions

Fixes #13603 

This PR inserts media captions defined in the site MediaLibrary when adding image blocks on the Gutenberg editor and on the Aztec editor.

To test:
 - Start the app
 - Start a new post in GB
 - Add an image block
 - Add an image of the WP media library that has a caption defined ( you can define it on the Media section of the site)
 - Check that the image is added and the caption is set
 - Do the same test using Aztec

PR submission checklist:
